clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -452,11 +452,12 @@ private boolean maybeSendTransactionalRequest(long now) {
                 if (targetNode != null) {
                     if (nextRequestHandler.isRetry())
                         time.sleep(nextRequestHandler.retryBackoffMs());
+                    long currentTimeMs = time.milliseconds();
```

Comment:
Hmm... Seems `now` is no longer used anymore?

Reply:
Ack. Updated the PR.